app/services/loader.js

```diff
@@ -13,7 +13,7 @@ export default Service.extend({
     if (config.isExternal) {
       options.url = url;
     } else {
-      options.url = config.withoutPrefix ? `${adapter.host}/${url}` : `${adapter.urlPrefix()}${url}`;
+      options.url = config.withoutPrefix ? `${adapter.host}${url}` : `${adapter.urlPrefix()}${url}`;
```

Comment:
@sumedh123 you're code 

```
      if (url.split('/')[0] === '') {
        url = url.split('/').splice(1).join('/');
      }
```

Itself is wrong. Did you try running it and seeing ? It is actually removing the leading slash. 

A very very simple code such as 

```js
url = `${url[0] !== '/' ? '/' :  ''}${url}`
```

will do the trick. Why do you have to split and splice and join and end up getting a wrong result ? In many cases, a simple answer is usually the best.